usr/bin/i2pbrowser

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+## Copyright (C) 2018 - 2018 ENCRYPTED SUPPORT LP <adrelanos@riseup.net>
+## See the file COPYING for copying conditions.
+
+export SCRIPTNAME="$(basename "$BASH_SOURCE")"
+
+export IDENTIFIER="i2pbrowser"
+
+ICON="/usr/share/icons/anon-icon-pack/tbupdate.ico"
+
+if [ ! -d "$HOME/.i2pb/" ]; then
+    echo "Error, i2p browser does not appear to be installed(run update-i2pbrowser)"
```

Comment:
This could use a GUI error message. `msgcollector`?

On the other hand, would not `/usr/bin/torbrowser` handle this already?

Reply:
Yes and no, since it uses the ~/.i2pb directory to store the profile, if it doesn't exist then the following cp -r (--no-clobber) won't have anywhere to put it. Easiest solution is just to run update-i2pbrowser in that conditional instead of exiting.